docs/CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 Jupytext ChangeLog
 ==================
 
+1.16.0-dev (2023-10-??)
```

Comment:
Can you move this to top-level Changelog.md and use `include` to link to it? [Here](https://github.com/scikit-build/scikit-build-core/blob/4697f07c72b04232f98fc4f40fdec4d201174977/docs/index.md?plain=1#L9)'s an example.

The idea is to have it in [keep-a-changelog](https://keepachangelog.com/en/1.1.0/) form so it is easier to incrementally add changelog

Reply:
Oh great! Actually I could use that for the `README.md` too (`index.md` is almost a verbatim copy of `../README.md`)